pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.github.han-yaeger</groupId>
     <artifactId>yaeger</artifactId>
-    <version>2020.2021-beta6-SNAPSHOT</version>
+    <version>2020.2021-beta7-SNAPSHOT</version>
```

Comment:
Leave the version at beta6-SNAPSHOT. With these last few bugs fixed, we can make a new release, after which we move on to beta7

Reply:
I changed the version locally so I could test the showcase againt my local version of Yaeger, I’ve reverted this